### PR TITLE
feat: Daisy UI (Delegate & Vote)

### PR DESCRIPTION
**Pull request by Senna46**
Support for Daisy UI
- Validators list
screencapture-localhost-4200-portal-utilities-delegate-validators-2023-03-31-21_26_22

- Validator detail
screencapture-localhost-4200-portal-utilities-delegate-validators-ununifivaloper1wd3g35hec5m6n4qfwaydndzelsuaafzjk98u3z-2023-03-31-21_26_33

- Proposals list
screencapture-localhost-4200-portal-utilities-vote-proposals-2023-03-31-21_27_02

- Proposal detail
screencapture-localhost-4200-portal-utilities-vote-proposals-2-2023-03-31-21_27_15

### CODE REVIEW

**Comment by Senna46** [comment]
@KimuraYu45z Plz review.

File: projects/portal/src/app/views/delegate/validators/validators.component.html [hunk 1]
```diff
@@ -56,57 +59,66 @@ <h2 class="mb-0">Delegations</h2>
...
-</ng-template>
-<ng-template #emptyUnbonding>
-  <p>There is no unbonding delegations.</p>
+      </tbody>
+    </table>
+  </div>
 </ng-template>
+<ng-template #emptyUnbonding></ng-template>
 
 <div class="flex flex-row items-end mb-2">
```
Comment (KimuraYu45z) [inline projects/portal/src/app/views/delegate/validators/validators.component.html:107]: shouldn't be items-end, and mb-2 and mb of h2 should b eliminated in design principle

**Review by KimuraYu45z — CHANGES_REQUESTED** [review]

File: projects/portal/src/app/views/delegate/validators/validators.component.html (hunk 1 above)
Comment (Senna46) [reply projects/portal/src/app/views/delegate/validators/validators.component.html]: This PR updated.
but t is present in many components. Should I remove them?

Comment (KimuraYu45z) [reply projects/portal/src/app/views/delegate/validators/validators.component.html]: what is t?

Comment (KimuraYu45z) [reply projects/portal/src/app/views/delegate/validators/validators.component.html]: so you mean this?

let's do later

**Review by KimuraYu45z — APPROVED** [review]

**Pull request merged**